app/routes/podcasts/components/Podcast.js

```diff
@@ -33,7 +33,7 @@ class Podcast extends Component<Props, State> {
   };
 
   render() {
-    const CLIENT_ID = 'E8IqLGTYxHll6SyaM7LKrMzKveWkcrjg';
+    const CLIENT_ID = 'WOwjzZGoYrSrOOAILojG4miJtS4pZjSg';
```

Comment:
This came in due to https://github.com/webkom/lego-webapp/pull/1583

I'm removing it as it actually works, we can close the PR above instead.